src/env.rs

```diff
@@ -15,6 +16,8 @@ pub static DUCKDB_CONNECTION_CACHE: PgLwLock<
     heapless::FnvIndexMap<u32, DuckdbConnection, MAX_CONNECTIONS>,
 > = PgLwLock::new();
 
+pub static mut DUCKDB_CONNECTION_LRU: PgLwLock<VecDeque<u32>> = PgLwLock::new();
```

Comment:
Nice implementation! I believe we want to use `heapless::VecDeque` since Postgres shared memory doesn't support dynamic memory allocation. 

https://github.com/pgcentralfoundation/pgrx/tree/develop/pgrx-examples/shmem

Reply:
Great catch, I will fix it.